quicklens/src/test/scala-3/com/softwaremill/quicklens/test/HugeModifyTest.scala

```diff
@@ -0,0 +1,66 @@
+package com.softwaremill.quicklens.test
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import com.softwaremill.quicklens._
+
+class HugeModifyTest extends AnyFlatSpec with Matchers {
```

Comment:
will this test also work for scala 2?

Reply:
It should. Should I move it?

Reply:
Well if it passes on both implementations, always good to have it being run for both scala 2 and scala 3 :)